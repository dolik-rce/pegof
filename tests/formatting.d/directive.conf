input formatting.d/directive.peg
