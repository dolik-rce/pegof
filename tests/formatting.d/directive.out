%prefix "prefix"

%value "value"

%auxil "auxil"

%header {
    #define TEST 1
}

%source { source }

%common { common }

%earlyheader { earlyheader }

%earlysource { earlysource }

%earlycommon {
    earlycommon
    multiple
    lines
}

%import "../formatting.d/expand.peg"
